R/snapshot.R

```diff
@@ -3,39 +3,33 @@
 # can be scoped to FALSE when hashing is not necessary
 `_renv_snapshot_hash` <- TRUE
 
-#' Snapshot a Project
+#' Record current state of the project library in the lockfile
 #'
-#' Call `snapshot()` to create a **lockfile** capturing the state of a project's
-#' \R package dependencies. The lockfile can be used to later restore these
-#' project's dependencies as required.
+#' @description
+#' Call `renv::snapshot()` to update a [lockfile][lockfiles] with the current
+#' state of dependences in the project library. The lockfile can be used to
+#' later [restore] these dependencies as required.
 #'
-#' See the [lockfile] documentation for more details on the structure of a
-#' lockfile.
+#' It's also possible to call `renv::snapshot()` with a non-`renv` project,
+#' in which case it will record the current state of dependencies in the
+#' system library. This makes it possible to [restore] the current packages,
```

Comment:
```suggestion
#' current library paths. This makes it possible to [restore] the current packages,
```

since "system library" means `.Library` in a technical sense.